build/yaml/runUnitTests-CI.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+- task: UseDotNet@2
+  displayName: 'Use .Net Core sdk 3.1.x'
+  inputs:
+    version: 3.1.x
+
+- task: DotNetCoreCLI@2
+  displayName: 'Build'
+  inputs:
+    command: build
+    publishWebProjects: false
+    projects: 'Tests/TranscriptTestRunnerTests/TranscriptTestRunnerTests.csproj'
+    arguments: '-v n --configuration $(BuildConfiguration)'
+
+- task: DotNetCoreCLI@2
+  displayName: 'DotNet Test'
+  inputs:
+    command: test
+    projects: 'Tests/TranscriptTestRunnerTests/TranscriptTestRunnerTests.csproj'
+    arguments: "-v n --configuration $(BuildConfiguration) --no-build --no-restore"
```

Comment:
Can we add `--collect "Code coverage"` so we can also capture code coverage details for the tests? 

See `publishTestResults` in [DotNetCoreCLI@2](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops) for more details
![image](https://user-images.githubusercontent.com/12264946/114644027-bc245780-9ca4-11eb-8baa-4792ba8d0696.png)

Reply:
Feedback applied. Thanks!